monkey/monkey_island/cc/models/monkey.py

```diff
@@ -2,23 +2,16 @@
 Define a Document Schema for the Monkey document.
 
 """
-import ring
 from mongoengine import (
-    BooleanField,
     DateTimeField,
     Document,
-    DoesNotExist,
     DynamicField,
     FloatField,
     ListField,
     ReferenceField,
     StringField,
 )
 
-from common.network.network_utils import get_my_ip_addresses_legacy
-from monkey_island.cc.models.monkey_ttl import MonkeyTtl, create_monkey_ttl_document
-from monkey_island.cc.server_utils.consts import DEFAULT_MONKEY_TTL_EXPIRY_DURATION_IN_SECONDS
-
 
 class Monkey(Document):
```

Comment:
This whole file also should go. Instead of `get_latest_modifytime` we can just fetch the timestamp of the latest event

Reply:
I am working on to see do we actually need the `modifytime`.

Reply:
Hmmm, maybe not. Removing `Monkey` would require more work, but we can certainly replace the `get_latest_modifytime`

Reply:
I'd rather not touch that code in this PR since Ilija is working on it as part of #2507.